edit-post/components/header/publish-sidebar-toggle/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress Dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { MenuItem } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
+import { withSelect, withDispatch } from '@wordpress/data';
+
+const PublishSidebarToggle = function( { onToggle, isEnabled } ) {
+	return (
+		<MenuItem
+			icon={ isEnabled && 'yes' }
```

Comment:
The icon here should be `'yes'`? That seems odd to me.

Reply:
That's how the [MenuItem](https://github.com/WordPress/gutenberg/tree/2d1e3e055c31a2df03aec83ce940a5dfe17447c6/packages/components/src/menu-item) works, it uses the [yes dashicon](https://developer.wordpress.org/resource/dashicons/#yes).

Reply:
Ideally this could switch to using the Material tick instead of the Dashicons tick, but that's obviously a bigger change that's out-of-scope for this PR.